pandas/tests/test_resample.py

```diff
@@ -2165,6 +2165,14 @@ def test_resample_datetime_values(self):
         res = df['timestamp'].resample('2D').first()
         tm.assert_series_equal(res, exp)
 
+    def test_resample_apply_with_additional_args(self):
+        def f(data, add_arg):
```

Comment:
added a comment referencing the github issue.

Reply:
Which issue?

Reply:
#14615